design-tokens-manager/src/routes/Scaler/components/ScaleLayout.jsx

```diff
@@ -0,0 +1,15 @@
+import { Box } from 'grommet';
+
+export const ScaleLayout = ({ ...rest }) => {
+  return (
+    <Box
+      direction="row"
+      align="end"
+      alignSelf="start"
+      cssGap
```

Comment:
Should we set `cssGap` true for the whole app? https://v2.grommet.io/grommet#options